app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityPresenter.kt

```diff
@@ -94,10 +94,7 @@ class AdministratorControlsActivityPresenter @Inject constructor(
       ) as AdministratorControlsFragment?
   }
 
-  /**
-   * Loads the profile list fragment as the [AdministratorControlsActivity] is
-   * started in multipane tablet mode.
-   */
+  /** Loads the profile list fragment as the [AdministratorControlsActivity] is started in multipane tablet mode. */
```

Comment:
This is going in the other direction. :) KDocs need to fit within the 100 character line limit, but they should take up as much space within that limit before wrapping.